### PR TITLE
Blazor integration for anchor menu item

### PR DESCRIPTION
**Pull request by m-akinc**
# Pull Request

## 🤨 Rationale

Blazor support for anchor menu item component

## 👩‍💻 Implementation

Followed standard process. Includes test, update to example app, and update to README.

## 🧪 Testing

Tested via example app

## ✅ Checklist

- [x] I have updated the project documentation to reflect my changes or determined no changes are needed.

### CODE REVIEW

File: packages/nimble-blazor/Tests/NimbleBlazor.Tests/Unit/Components/NimbleAnchorMenuItemTests.cs [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+
+namespace NimbleBlazor.Tests.Unit.Components;
+
+/// <summary>
+/// Tests for <see cref="NimbleAnchorMenuItem"/>.
+/// </summary>
+public class NimbleAnchorMenuItemTests
+{
+    [Fact]
+    public void NimbleAnchorMenuItem_Render_HasAnchorMenuItemMarkup()
```
Comment (atmgrifter00) [inline packages/nimble-blazor/Tests/NimbleBlazor.Tests/Unit/Components/NimbleAnchorMenuItemTests.cs:12]: Any reason to not follow other test examples (like the [Combobox](https://github.com/ni/nimble/blob/f60dedf600147b16916f876b749a4e019c2a6308/packages/nimble-blazor/Tests/NimbleBlazor.Tests/Unit/Components/NimbleComboboxTests.cs#L11)), and test _all_ attributes? I think the components that don't do that are some of the earlier ones.

Comment (m-akinc) [reply packages/nimble-blazor/Tests/NimbleBlazor.Tests/Unit/Components/NimbleAnchorMenuItemTests.cs:14]: No problem. Tests added.

**Review by jattasNI — COMMENTED** [review]
I defer to Blazor experts on this PR

File: packages/nimble-blazor/NimbleBlazor/Components/NimbleAnchorMenuItem.razor.cs [hunk 2]
```diff
@@ -0,0 +1,66 @@
+﻿using Microsoft.AspNetCore.Components;
+
+namespace NimbleBlazor;
+
+public partial class NimbleAnchorMenuItem : ComponentBase
```
Comment (rajsite) [inline packages/nimble-blazor/NimbleBlazor/Components/NimbleAnchorMenuItem.razor.cs:5]: Can we make a NimbleAnchorBase class like we did in Angular and Components so we can avoid duplicating all these properties and docs?

File: packages/nimble-blazor/Tests/NimbleBlazor.Tests/Unit/Components/NimbleAnchorMenuItemTests.cs [hunk 3]
```diff
@@ -0,0 +1,95 @@
...
+using System.Linq.Expressions;
+using Bunit;
+using Xunit;
+
+namespace NimbleBlazor.Tests.Unit.Components;
+
+/// <summary>
+/// Tests for <see cref="NimbleAnchorMenuItem"/>.
+/// </summary>
+public class NimbleAnchorMenuItemTests
```
Comment (rajsite) [inline packages/nimble-blazor/Tests/NimbleBlazor.Tests/Unit/Components/NimbleAnchorMenuItemTests.cs:11]: Can we make an AnchorBase Xunit theory test or w/e it's called to make sure we are exercising all the shared link properties in one spot?

**Review by msmithNI — APPROVED** [review]

File: packages/nimble-blazor/NimbleBlazor/Components/NimbleAnchorMenuItem.razor.cs (hunk 2 above)
Comment (m-akinc) [reply packages/nimble-blazor/NimbleBlazor/Components/NimbleAnchorMenuItem.razor.cs]: Done.

**Comment by rajsite** [comment]
There have been significant changes so pinging a blazor expert to re-review jic @atmgrifter00

**Review by atmgrifter00 — APPROVED** [review]

**Pull request merged**